### PR TITLE
Buttons: add more examples

### LINKED ISSUE #31821: add a button to home assistant
### Feedback

1. The description displays "select the three dots menu, then select Take control". I was offered 4 options, but not one for "Take Control".
2. The description display "". I was not presented with a "card picker". I was presented with a tabbed choice of "BY CARD" or "BY ENTITY". I chose "BY CARD".
3.  I was able to choose the entity I wanted. 
4. Now I want to implement some activity for when the button is actioned. I need help to decide whether i use an "Automation" or a "Script" or something else!
5. The guidance then states "...when you use YAML mode or just prefer to use YAML...". This piece of information is meaningless to me. I need more help.

### URL

https://www.home-assistant.io/dashboards/button/

### Version

2024.3.0

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Buttons: add more examples
- To give some inspiration on using buttons with scripts
- addresses feedback from #31821

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #31821

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
  - Updated button card documentation to include a reference to a script service.
  - Added an example of 4 buttons on a vertical stack card with detailed actions.
  - Provided guidance and example configurations for creating scripts associated with buttons, including actions like Netflix, YouTube, wake on LAN, and sleep.

### CODE REVIEW

File: source/_dashboards/button.markdown [hunk 1]
```diff
@@ -141,7 +146,58 @@ tap_action:
...
+```yaml
+cards:
+  - entities:
+      - entity: input_select.living_room_scene
+        name: Scene
+    show_header_toggle: false
+    type: entities
+  - type: horizontal-stack
+    cards:
+      - show_name: true
```
Comment (klaasnicolaas) [inline source/_dashboards/button.markdown]: I would start first with `- entity: ...`

**Comment by frenck** [comment]
I've marked this PR, as changes are requested that need to be processed.
Please un-draft it once it is ready for review again by clicking the **"Ready for review"** button.

Thanks! 👍 

../Frenck

[_Learn more about our pull request process._](https://developers.home-assistant.io/docs/review-process#prs-are-being-drafted-when-changes-are-needed)

File: source/_dashboards/button.markdown [hunk 2]
```diff
@@ -141,7 +146,60 @@ tap_action:
...
+Example of 4 buttons on a vertical stack card:
+
 <p class='img'>
-<img src='/images/dashboards/entity_button_complex_card.png' alt='Screenshot of the Button card with script service'>
-Screenshot of the button card with script service.
+<img src='/images/dashboards/buttons_on_vertical_stack_card.png' alt='Screenshot of a vertical stack card with 4 buttons and an entity selector'>
+Screenshot of a vertical stack card with 4 buttons and an entity selector.
 </p>
+
+The image shows a vertical stack card with 4 buttons arranged in a horizontal stack card and an entity selector. The buttons use the toggle action to run a script, for example the netflix script which starts up the TV and opens Netflix. To learn how to create scripts, refer to [scripts](/docs/scripts/).
```
Comment (c0ffeeca7) [inline source/_dashboards/button.markdown]:
```suggestion
The image shows a vertical stack card with 4 buttons arranged in a horizontal stack card and an entity selector. The buttons use the toggle action to run a script, for example, the Netflix script, which starts up the TV and opens Netflix. To learn how to create scripts, refer to [scripts](/docs/scripts/).
```

File: source/_dashboards/button.markdown [hunk 3]
```diff
@@ -141,7 +146,60 @@ tap_action:
...
+Screenshot of a vertical stack card with 4 buttons and an entity selector.
 </p>
+
+The image shows a vertical stack card with 4 buttons arranged in a horizontal stack card and an entity selector. The buttons use the toggle action to run a script, for example, the Netflix script, which starts up the TV and opens Netflix. To learn how to create scripts, refer to [scripts](/docs/scripts/).
+
+```yaml
+cards:
+  - entities:
+      - entity: input_select.living_room_scene
+        name: Scene
```
Comment (frenck) [inline source/_dashboards/button.markdown:162]:
```suggestion
     - entity: input_select.living_room_scene
       name: Scene
```

File: source/_dashboards/button.markdown [hunk 4]
```diff
@@ -141,7 +146,60 @@ tap_action:
...
+Screenshot of a vertical stack card with 4 buttons and an entity selector.
 </p>
+
+The image shows a vertical stack card with 4 buttons arranged in a horizontal stack card and an entity selector. The buttons use the toggle action to run a script, for example, the Netflix script, which starts up the TV and opens Netflix. To learn how to create scripts, refer to [scripts](/docs/scripts/).
+
+```yaml
+cards:
+  - entities:
+     - entity: input_select.living_room_scene
+       name: Scene
```
Comment (frenck) [inline source/_dashboards/button.markdown]:
```suggestion
      - entity: input_select.living_room_scene
        name: Scene
```

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**